src/NightwatchExt/core.ts

```diff
@@ -39,17 +42,40 @@ export class NightwatchExt {
 
   // The ability to show fails in the problems section
   private failDiagnostics: vsCodeTypes.DiagnosticCollection;
+  public quickSettingPanel: QuickSettingPanel;
+  public nightwatchSettings: Settings;
+  public environmentsPanel: EnvironmentsPanel;
 
   constructor(
     private _vscode: vsCodeTypes.VSCode,
```

Comment:
why is only one private property named with a underscore in the beggining? 

```suggestion
    private vscode: vsCodeTypes.VSCode,
```

Reply:
These kinds of issues exist in multiple places in the project. We can fix this in a separate PR for refactoring. Ignoring this comment for now